README.md

```diff
@@ -1 +1,44 @@
-# lugbench-api
\ No newline at end of file
+# LugBench-API
+
+# Prerequisites
+
+Program | Version
+------- | -----------
+node    | >= 6.10x
+npm     | >= 4.5x
+
+# Clone the repository
+
+```shell=
+git clone git@github.com:Lugdunum3D/LugBench-API.git
+```
+
+# Installation
+
+This command line below will install all dependencies you need to run the project.
+```shell=
+npm install
+```
+
+# Running the API in *http://localhost:5000*
+
+```shell=
+npm start
+```
+
+:::info
+The default port is 5000. If you want to set a different one for the API to listen, set a environment variable system like this:
+
+```shell=
+export PORT=3000
+```
+:::
+
+
+# Routes
```

Comment:
It's great to have all the routes, but can't we provide also the content of the JSON? With a JSON Schema for example.